### DISCUSSION TITLE
Isn't the developer account free?

### DISCUSSION THREAD

**Discussion by xheox** [amzn/selling-partner-api-models#3680] [General]
We are not going to sell anything on Amazon, just create public apps (and may be private too).
One of the developer account setup requirements is to have a professional seller account.
I went to set up the account and it is asking me for credit card info for a "Monthly subscription fee".
We will use the account only for developer purpose so I am not sure if this is really required. 
can anyone confirm please ?
Thanks

**Comment by leonardonakagawa** [comment]
Hello @xheox,

Thank you for reaching out regarding Developer Account.

Yes, only Professional Selling Accounts can register to develop or integrate with Selling Partner API. Individual accounts are not eligible. You can upgrade your account to a professional plan at any time. You can view your selling plan type and the marketplace information under Your Services in [Account info](https://sellercentral.amazon.com//hz/sc/account-information). Vendors can register as private developers on Vendor Central. For more information, refer to [Registering as a developer](https://developer-docs.amazon.com/sp-api/docs/registering-as-a-developer).

You can find details regarding this in our [Do I need a Professional Selling Account to register as a Selling Partner API developer?](https://developer-docs.amazon.com/sp-api/docs/sp-api-general-faqs#do-i-need-a-professional-selling-account-to-register-as-a-selling-partner-api-developer).
If the information available in the documentation link provided above does not fully resolve your inquiry, please [open a support case](https://developer.amazonservices.com/support) with us.

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support
